### ISSUE TITLE
lost support for armbian arm64

### ISSUE THREAD

**Issue by ggmartins** [saltstack/salt-bootstrap#1138]
### Description of Issue/Question
I'm pretty sure this **used to work**, but right now I'm getting this:
root@odroidc2:~# sh bootstrap-salt.sh -r -P git v2016.11.1
...
Ign http://apt.armbian.com jessie/utils Translation-en_US
Ign http://apt.armbian.com jessie/utils Translation-en
Ign http://apt.armbian.com jessie/utils Translation-en_US.UTF-8
Fetched 195 kB in 7s (27.1 kB/s)
Reading package lists...
 * ERROR: repo.saltstack.com doesn't have packages for your system architecture: **arm64**.
 * ERROR: Failed to run install_debian_8_git_deps()!!!

Any chance or advice on how to recover the old state/ability to install SS on odroid-c2/arm64 ? thanks a ton!

### Setup
Device flashed with OS Armbian_5.25_Odroidc2_Debian_jessie_default_3.14.79.img

### Steps to Reproduce Issue
sh bootstrap-salt.sh git v2016.11.5
sh bootstrap-salt.sh -r -P git v2016.11.1

### Versions and Systems
sh bootstrap-salt.sh -v 
bootstrap-salt.sh -- Version 2017.08.17
uname -a
Linux odroidc2 3.14.79-odroidc2 #32 SMP PREEMPT Wed Feb 1 22:28:54 CET 2017 aarch64 GNU/Linux

**Comment by vutny** [comment]
@ggmartins Thanks for reporting! That definitely worked before. It seems easy to fix, I'll try to get on this.

**Comment by rallytime** [comment]
@vutny Thanks for jumping on fixing this. I was just about to comment here and ask if you could look into it. :)

**Comment by vutny** [comment]
@ggmartins I've attempted to fix this in PR #1141. Please give it a try.

**Comment by ggmartins** [comment]
I've downloaded (link below) and installed minion via sh _bootstrap-salt.sh -r -P git v2016.11.5_
https://raw.githubusercontent.com/vutny/salt-bootstrap/f283f92563f8b8cafdb9f887f6730ed02e58fba2/bootstrap-salt.sh
It seems to be running, but it didn't request access to the master I set. I doubled checked cfg multiple times, restarted _(salt-call --local service.restart salt-minion)_ , rebooted and did't work.
Logs are showing:
**DNS lookup or connection check of 'salt' failed.**
**Master hostname: 'salt' not found or not responsive. Retrying in 30 seconds**
It's like the cfg I created hasn't been considered/applied. Anyways, I'll triple check everything.
